Resources/Prototypes/_Stories/Entities/Stations/prison.yml

```
@@ -0,0 +1,46 @@
+- type: entity
+  id: PrisonStation
```

```suggestion
  id: STPrisonStation
```
А так же в прототипе карты тюрьмы надо поменять